homeassistant/components/nina/binary_sensor.py

```diff
@@ -69,7 +75,10 @@ def __init__(
 
         self._attr_name = f"Warning: {region_name} {slot_id}"
         self._attr_unique_id = f"{region}-{slot_id}"
-        self._attr_device_class = BinarySensorDeviceClass.SAFETY
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, config_entry.entry_id)},
+            manufacturer="NINA",
```

Comment:
Isn't this a service? Then we should set the `entry_type` to service.

https://developers.home-assistant.io/docs/device_registry_index#device-properties

Reply:
Right, addressed in https://github.com/home-assistant/core/pull/103367